src/common/logger.ts

```diff
@@ -1,6 +1,17 @@
 import * as bunyan from 'bunyan';
 import config = require('./config');
 
+if (process.env.LOG_LEVEL) {
+  try {
+    // Validate if LOG_LEVEL has valid bunyan logging level
+    bunyan.resolveLevel(process.env.LOG_LEVEL);
+  } catch {
+    delete process.env.LOG_LEVEL;
```

Comment:
I know it is a problematic context but can we log something here? so the customer knows what happened